includes/apple-push-api/class-mime-builder.php

```diff
@@ -76,7 +78,13 @@ public function add_metadata( $meta ) {
 	 * @access public
 	 */
 	public function add_json_string( $name, $filename, $content ) {
-		return $this->build_attachment( $name, $filename, $content, 'application/json' );
+		return $this->build_attachment(
+			$name,
+			$filename,
+			$content,
+			'application/json',
+			strlen( $content )
```

Comment:
if you're concerned about multi-byte chars, you could use[ `mb_strlen()`](http://php.net/manual/en/function.mb-strlen.php)